packages/circuit-ui/components/NotificationInline/NotificationInline.stories.tsx

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable array-callback-return */
```

Comment:
Let's not disable things for the entire file, but for specific lines (or sections) instead. It makes it more explicit